lab_1_keywords_tfidf/main.py

```diff
@@ -33,6 +44,18 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list
 
     In case of corrupt input arguments, None is returned
     """
+    if not (isistance(tokens, list) and isinstance(stop_words, list)):
+        return None
+    for stop_word in stop_words:
+        while stop_word in tokens:
+            tokens.remove(stop_word)
```

Comment:
consider creating a new list and updating it instead of modifying the one that you were given